portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/fileinstall/FileInstall.testcase

```diff
@@ -121,6 +154,158 @@ definition {
+				serviceAccessPolicyName = "SYSTEM_USER_PASSWORD_2",
+				urlsInclude = "/api/liferay/test2");
+		}
+	}
+
+	@description = "Verify the default factory config created by the OSGI configurator can be overriden."
+	@priority = "3"
+	test OverrideDefaultFactoryConfig {
+		property test.name.skip.portal.instance = "FileInstall#OverrideDefaultFactoryConfig";
+
+		task ("Given the default factory config created by the OSGI configurator is deleted") {
+			SystemSettings.openSystemSettingsAdmin();
+
+			SystemSettings.gotoConfiguration(
+				configurationCategory = "API Authentication",
+				configurationName = "Tunnel Authentication",
+				configurationScope = "System Scope");
+
+			LexiconEntry.gotoEntryMenuItemNoError(
+				menuItem = "Delete",
+				rowEntry = "/api/liferay/do");
+
+			LexiconEntry.viewNoEntry(rowEntry = "/api/liferay/do");
+		}
+
+		task ("When the default factory config is recreated via an OSGI config file") {
+			OSGiConfig.deployOSGiConfigFile(
+				osgiConfigFileName = "com.liferay.portal.security.auth.verifier.internal.tunnel.configuration.TunnelAuthVerifierConfiguration~default.config",
+				osgiConfigs = "enabled=&quot;false&quot;,hostsAllowed=&quot;127.0.0.1&quot;,serviceAccessPolicyName=&quot;SYSTEM_USER_PASSWORD_2&quot;,urlsIncludes=&quot;/api/liferay/test&quot;");
+		}
+
+		task ("Then the default factory configuration should be successfully re-added") {
+			Portlet.shutdownServer();
+
+			Portlet.startServer();
+
+			SystemSettings.openSystemSettingsAdmin();
+
+			SystemSettings.gotoConfiguration(
+				configurationCategory = "API Authentication",
+				configurationName = "Tunnel Authentication",
+				configurationScope = "System Scope");
+
+			LexiconEntry.gotoEntry(rowEntry = "/api/liferay/test");
+
+			APIAuthentication.assertConfigInSystemSettings(
+				enabled = "false",
+				hostsAllowed = "127.0.0.1",
+				serviceAccessPolicyName = "SYSTEM_USER_PASSWORD_2",
+				urlsInclude = "/api/liferay/test");
+		}
+	}
+
+	@description = "Verify when an existing factory config is renamed, the old entry is removed and the new one is added."
+	@priority = "3"
+	test RenameFactoryConfig {
+		property osgi.module.configuration.file.names = "com.liferay.portal.security.auth.verifier.internal.tunnel.configuration.TunnelAuthVerifierConfiguration~test.config";
+		property osgi.module.configurations = "enabled=&quot;true&quot;${line.separator}hostsAllowed=&quot;127.0.0.1,SERVER_IP&quot;${line.separator}serviceAccessPolicyName=&quot;SYSTEM_USER_PASSWORD&quot;${line.separator}urlsIncludes=&quot;/api/liferay/test&quot;";
+		property test.name.skip.portal.instance = "FileInstall#RenameFactoryConfig";
+
+		task ("When a factory configuration is renamed") {
+			var liferayHome = PropsUtil.get("liferay.home.dir.name");
+			var configName = "com.liferay.portal.security.auth.verifier.internal.tunnel.configuration.TunnelAuthVerifierConfiguration";
+
+			var configFile = "${liferayHome}/osgi/configs/${configName}~test.config";
```

Comment:
Try to avoid using names like test. For example in line 295, because the config file name is test, a test reader may confuse that path as being a static path instead of referencing the name of this configuration. Original/new Old/new are some examples of keywords used in integration tests for renaming